engine-tests/src/test/java/org/terasology/config/flexible/AutoConfigManagerTest.java

```diff
@@ -51,9 +37,9 @@
 
     @BeforeEach
     public void setUp() throws Exception {
-        final JavaArchive homeArchive = ShrinkWrap.create(JavaArchive.class);
-        final FileSystem vfs = ShrinkWrapFileSystems.newFileSystem(homeArchive);
-        PathManager.getInstance().useOverrideHomePath(vfs.getPath(""));
+        Path tempHome = Files.createTempDirectory("terasology-env", (FileAttribute<?>[]) null);
+        tempHome.toFile().deleteOnExit();
+        PathManager.getInstance().useOverrideHomePath(tempHome);
```

Comment:
suggestion: as this is a test class and not some abstract base class, this could use [`@TempDir`](https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory).